apps/emqx/src/emqx_listeners.erl

```diff
@@ -34,37 +34,34 @@
         , stop_listener/3
         , restart_listener/1
         , restart_listener/3
+        , has_listener_conf_by_type/1
         ]).
 
 -spec(list() -> [{ListenerId :: atom(), ListenerConf :: map()}]).
 list() ->
```

Comment:
what is the function of this list()?
It list the running listeners or just configured listeners?

We will have the following API to list running listeners for quic,
quicer:listeners().

Reply:
this lists all configured listeners. will add a `@doc`